packages/aws-lambda-otel-extension/opt/otel-extension/external/otel-payloads.js

```diff
@@ -169,14 +169,15 @@ const createLogPayload = (fun, logs) => {
   return logs.map((log) => {
     const split = (log.record || '').split('\t');
     return {
-      Timestamp: split[0] ? new Date(split[0]).getTime() : new Date().getTime(),
+      Timestamp: new Date(log.time).getTime(),
       Attributes: resourceAtt,
       Resource: metricsAtt,
       TraceId: spanData.traceId,
       SpanId: spanData.spanId,
       SeverityText: severityLevelNames.has(split[2]) ? split[2] : undefined,
       SeverityNumber: severityNumberMap[split[2]],
       Body: log.record || '',
+      order: process.hrtime.bigint().toString(),
```

Comment:
I assume we need that, as for logs for which timestamp is same, order gets broken, right?

Also, aren't we supposed here to follow some Otel spec, for generated payload?

Additionally `order` name (1) doesn't follow naming convention (it's not upper case), and it feels pretty vague (it's hard to assume what it's about just from name), maybe we can name it as

Wouldn't it be better to name it as e.g. `ReceptionHighResolutionTimestamp` ?

Reply:
>  I do like the name HighResolutionTimestamp to better fit in with the other attributes 

I meant _Reception_ as it's a timestamp when our extension processed that log, and it's not really a timestamp when log was produced (afaik in most cases it'll be 100ms+ delays, right?).

Anyway in such case maybe we can call it `ReceptionOrderId` (?) So we do not attach to fact that it's a timestamp, as _time of reception_ is not a key here, but _order_

Reply:
Okay thanks for the clarification. Yeah this makes sense to me 👍 I like `ReceptionOrderId`. I will make that change here.

Reply:
@medikoo actually after thinking on this a bit more I think the word `Reception` is kind of confusing. Maybe we just call it `ProcessOrderId` instead?

Reply:
Maybe `ProcessingOrderId` ? :) I wonder whether _process_ will not get confused with e.g. node.js process.

Still I leave it up to you

Reply:
I think I like `ProcessingOrderId` 👍 I think the `OrderId` part makes it distinct enough against the node.js process 😎

Reply:
I’m going to go ahead and merge this in since all the feedback has been addressed 😎